### ISSUE TITLE
Connect the new txtuml project - model - file wizards

### ISSUE THREAD

**Issue by devaigergely81** [ELTE-Soft/txtUML#504]
The usual workflow of a modeller is currently to execute these 3 wizards in a sequence in order to start working on a new model. Suggestion:
- When the "New txtUML project" wizard page is correctly filled, the user should have to options:
  - "Finish": The wizard ends and created the project as of today.
  - "Cancel": The wizard silently ends as of today.
  - "Next": The project is created, but the wizard continues with the "New model" wizard, where the source folder field could already be filled with the correct path.
 - When the "New model" wizard is completed, the user would again have the option to "Finish", "Cancel" or proceed with "Next" to the "New XtxtUML file" wizard. (This last option should only be present the syntax was selected to be X.)

Eclipse wizards support this kind of chaining, see for example similar wizards of Papyrus.

**Comment by djnemeth** [comment]
It would be even more convenient if we allowed going "Back" during the process to change already specified details. This would of course imply that we shouldn't generate anything until "Finish" is pressed.

@dobreffandras, do you think this is achievable with our wizard pages?

**Comment by dobreffandras** [comment]
Yes, I think all of the above mentioned requirements are achievable.
